doc/src/manual/parallel-computing.md

```diff
@@ -1099,11 +1099,17 @@ The [`launch()`](@ref) method is called asynchronously in a separate task. The t
 this task signals that all requested workers have been launched. Hence the [`launch()`](@ref)
 function MUST exit as soon as all the requested workers have been launched.
 
-Newly launched workers are connected to each other, and the master process, in an all-to-all manner.
-Specifying the command argument `--worker <cookie>` results in the launched processes initializing
-themselves as workers and connections being set up via TCP/IP sockets. Optionally, `--bind-to bind_addr[:port]`
-may also be specified to enable other workers to connect to it at the specified `bind_addr` and
-`port`. This is useful for multi-homed hosts.
+Newly launched workers are connected to each other, and the master process in an all-to-all manner.
+Specifying the command argument `--worker=<cookie>` results in the launched processes
```

Comment:
Maybe `--worker`/`--worker=<cookie>`, so that it is clear that it is an optional argument?
I have also seen `--worker[=<cookie>]` to denote optional arguments.